openfisca_aotearoa/variables/acts/social_security/12/childcare_subsidy.py

```diff
@@ -0,0 +1,64 @@
+
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class childcare_subsidy__is_a_dependant_child(Variable):
```

Comment:
We'd only need a special variable for this if childcare_subsidy has it's only unique defintion of what a dependent child it.

i think it should be the existing dependent child under the interpretation in the social security act.